package.json

```diff
@@ -76,6 +75,8 @@
     "@aws-sdk/client-sts": "^3.319.0",
     "@aws-sdk/types": "^3.257.0",
     "@chakra-ui/icons": "^2.0.17",
+    "@chakra-ui/react": "^2.6.1",
+    "@tinystacks/ops-core-widgets": "0.0.11",
```

Comment:
Do we need specific versions of these packages? Or can we keep these as peer dependencies?

Reply:
I think we can keep them as peer dependencies, I was getting errors locally for them not being installed but I think it's my problem. So let me update this